1-tolean.py

```diff
@@ -509,32 +524,33 @@ def print_as_lean(opt):
   out += "=>\n"
   out += to_str_prog(tgt, []) + "\n"
   out += "-/\n"
-  out += "open SSA EDSL in\n"
-  out += ("example : forall (w : Nat) ")
-  out += "(" + " ".join(src_state.constant_names + tgt_state.constant_names) + " : Nat)"
+  out += ("theorem alive_" + (sanitize_name(name)) + (" : forall (w : Nat) " if bitwidth == 'w' else ": forall "))
```

Comment:
Can we split this printing into a separate line?


```suggestion
  out += ("theorem alive_" + (sanitize_name(name)) + "forall ":
  if bitwidth == 'w': out +=  "(w : Nat)"
```